.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+version: 2.1
```

Comment:
I think it'd be preferable to stick to GitHub Actions since it's already in place.